src/bin/resource_tracing.rs

```diff
@@ -878,7 +983,54 @@ fn main()
 
             let mut room_locations = vec![];
             let mut room_removals = HashMap::new();
+            let mut door_locations = vec![];
+
+            let target_mapa_id = mapw.next().unwrap().into_owned();
+            let target_mapa = resources.iter()
+                .find(|res| res.fourcc() == b"MAPA".into() && res.file_id == target_mapa_id)
+                .unwrap().into_owned();
+            let mapa_id = target_mapa.file_id;
+
             for (layer_num, scly_layer) in scly.layers.iter().enumerate() {
+
+                // trace door resources //
+                for obj in scly_layer.objects.iter() {
+                    let obj = obj.into_owned();
+                    let door = if let Some(door) = obj.property_data.as_door() {
+                        door
+                    } else {
+                        continue
+                    };
+                    // Skip all doors that aren't openable.
+                    if door.ancs.file_id != 0x26886945 && door.ancs.file_id != 0xFAFB5784 { continue; };
+
+                    // Skip all problematic doors (all in frigate intro level) //
+                    if  obj.instance_id == 0x70009 ||
```

Comment:
Split this into an array and an `if !PROBLEMATIC_DOORS.contains(&obj.instance_id) {`